### PR TITLE
Add UP as a default option

### LINKED ISSUE #85: authenticatorGetInfo should use defaults for options
Thanks for this tooI. I had an error indicating `Check failed: condition Failed critical condition: User presence support expected.` because in the authenticatorGetInfo response the authenticator is not including the `up` flag.  However, according to the specification (and the conformance tests) _When an option is not present, the default is applied per table below_ so this option being absent is a valid configuration and the tool should instead use the default.

Do you agree to add this fix to the backlog?

### PR DESCRIPTION
**Pull request by kaczmarczyck**
Fixes #85 

@cheburashka Can you please test and confirm it works with this PR? I didn't have a security key that doesn't report the UP capability, so I didn't notice. I tested this PR with an OpenSK that doesn't report `UP` and it seems to work.

### CODE REVIEW

File: src/device_tracker.cc [hunk 1]
```diff
@@ -105,6 +105,14 @@ void DeviceTracker::Initialize(const cbor::Value::ArrayValue& versions,
       }
     }
   }
+
+  std::vector<std::string> default_true_options = {"up"};
+  for (const std::string& option : default_true_options) {
+    auto iter = options.find(cbor::Value(option));
+    if (iter == options.end()) {
```
Comment (ia0) [inline src/device_tracker.cc:112]: There seems to be a `contains` method, so `!options.contains(cbor::Value(option))` should be an equivalent condition.

**Review by ia0 — APPROVED** [review]

File: src/device_tracker.cc (hunk 1 above)
Comment (kaczmarczyck) [reply src/device_tracker.cc:112]: `std::map` has `contains` since C++20, but we use C++17. So this should be the best we can do?

Comment (ia0) [reply src/device_tracker.cc:112]: Ah ok it's `std::map`. I don't know where to get this information. I just tried to Google `cbor::Value::MapValue` and found [this definition](https://chromium.googlesource.com/chromium/src/+/master/components/cbor/values.h#96) which seems to have [a `contains` method](https://chromium.googlesource.com/chromium/src/+/master/base/containers/flat_map.h#157). Sounds good if it doesn't compile with `contains`.

Comment (kaczmarczyck) [reply src/device_tracker.cc:112]: When I ported the CBOR parser out of Chromium, I replaced some data structures with standard replacements. You could check `third_party/`. The compiler agrees there is no `contains`.

(btw: The port was done when we still used C++11, so I think there are some possible improvements there. But unrelated and it shouldn't be important right now.)

Comment (ia0) [reply src/device_tracker.cc:112]: Oh I see, I missed that `third_party` was not "generated" :D Makes sense, thanks for the info, now it's simpler to know where to check.

**Pull request merged**